tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1288,6 +1297,9 @@ def build(self, input_fn):
     if not input_fn:
       raise RuntimeError("input_fn is None. Method build() needs input_fn "
                          "to be specified in order to build TensorRT engines")
+    if self._need_calibration and self._need_trt_profiles() and \
```

Comment:
can we also add document to the "build" function, to describe the delay calibration step here?

One thing that doesn't look nice is that the input_fn provided here is not used if calibration is needed. 

Is this scenario possible: First convert the graph, saved the converted graph. Then later on, create another converter and build the graph. In that case, are we able to "carry" the calibration_input_fn over?